src/extension.ts

```diff
@@ -1197,6 +1198,48 @@ export async function activate(context: vscode.ExtensionContext) {
     return result.trim();
   });
 
+  registerIDFCommand("espIdf.selectOpenOcdConfigFiles", async () => {
+    try {
+      const boards = await getBoards();
+      const choices = boards.map((b) => {
+        return {
+          description: `${b.description} (${b.configFiles.join(",")})`,
+          label: b.name,
+          target: b,
+        };
+      });
+      const selectOpenOCdConfigsMsg = locDic.localize(
+        "extension.enterOpenOcdConfigMessage",
+        "Enter OpenOCD Configuration File Paths list"
+      );
+      const selectedBoard = await vscode.window.showQuickPick(choices, {
+        placeHolder: selectOpenOCdConfigsMsg,
+      });
+      if (!selectedBoard) {
```

Comment:
When User select a board, I think we can show a info message that board selection is successful